monero/wallet_client.go

```diff
@@ -591,10 +591,10 @@ func getWalletRPCFlags(conf *WalletClientConf) []string {
 	return args
 }
 
-// getFreePort returns an OS allocated and immediately freed port. There is nothing preventing
+// getFreeTCPPort returns an OS allocated and immediately freed port. There is nothing preventing
 // something else on the system from using the port before the caller has a chance, but OS
 // allocated ports are randomised to make the risk negligible.
-func getFreePort() (uint, error) {
+func getFreeTCPPort() (uint, error) {
```

Comment:
Since the code now uses 2 different port types, I wanted to preemptively avoid confusion.